backend/tests/helpers/UserHelper.test.js

```diff
@@ -2,7 +2,7 @@ const { expect } = require( 'chai' );
 const { getUserListFromSessions } = require( '../../src/helpers/helpers' );
 
 describe( 'getUsers', function() {
```

Comment:
() => {}

Reply:
Also, change the file name. Remove the `.test` extension and fix the `test: //` regexp in the `package.json`

Reply:
Done.